### PR TITLE
Add script for jaktskildringar.

### PR DESCRIPTION
**Pull request by jannistsiroyannis**

### CODE REVIEW

**Review by olovy — COMMENTED** [review]
LXL-3536 was a bit vague but as I understood it we also want to change subject: sao/Jaktskildringar to genreForm: saogf/Jaktskildringar?

gernreForm: sao/Jaktskildringar  are only 5 records [in PROD](https://libris.kb.se/katalogisering/search/libris?o=https%3A%2F%2Fid.kb.se%2Fterm%2Fsao%2FJaktskildringar) while  subject: sao/Jaktskildringar to genreForm are 79. 

ComplexSubject termComponent (45) I guess we can't do nothing about.

**Comment by jannistsiroyannis** [comment]
:+1: I've checked with Viktoria and its _only_ subject we're supposed to change. I'll fix it.

**Review by klngwll — COMMENTED** [review]
I do agree the description of the issue is a bit vague. In general we don't have GenreForm terms in subjects so I would have guessed also the relation should have changed to genreForm.

**Review by klngwll — CHANGES_REQUESTED** [review]
So what I figured out is that the relationship for these should be changed from subject to genreForm as well.

**Comment by jannistsiroyannis** [comment]
> So what I figured out is that the relationship for these should be changed from subject to genreForm as well.

Better now? :)

File: whelktool/scripts/cleanups/2021/04/lxl-3536-jaktskildringar.groovy [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    if (containedBadLink) {
+
+        // Make sure genreForm is a list
+        if (work.genreForm == null)
+            work.put("genreForm", [])
+        else if ( !(work.genreForm instanceof List) )
+            work.put("genreForm", [work.genreForm])
+
+        // Add the correct link
+        work.genreForm.add(["id" : newLinkTarget])
```
Comment (olovy) [inline whelktool/scripts/cleanups/2021/04/lxl-3536-jaktskildringar.groovy:43]:
```suggestion
        work.genreForm.add(["@id" : newLinkTarget])
```

**Review by olovy — COMMENTED** [review]
Almost!

The rest LGTM

**Review by klngwll — APPROVED** [review]
LGTM

**Comment by jannistsiroyannis** [comment]
> Almost!
> 
> The rest LGTM

Good catch! :)

**Pull request merged**